src/nvim/terminal.c

```diff
@@ -313,6 +313,10 @@ void terminal_resize(Terminal *term, uint16_t width, uint16_t height)
     return;
   }
 
+  if (height == 0 || width == 0) {
+    return;
+  }
```

Comment:
Should this maybe be above the `FOR_ALL_WINDOWS_IN_TAB` block? Or maybe we should be checking this there to make sure that `MIN` isn't the problem.